bedrock/firefox/templates/firefox/tracking-protection-tour/variation-1.html

```diff
@@ -32,7 +38,11 @@ <h1>{{ _('Content Blocking') }}</h1>
 
 {% block panel_2 %}
   <h2>{{ _('Differences to expect') }}</h2>
+  {% if l10n_has_tag('remove_fastblock_1499589') %}
+  <p>{{ _('Content blocking keep parts of pages or entire pages from loading.') }}</p>
```

Comment:
Completely missed this :-\

A localizer noticed that "keep" should "keeps", so I checked the copy, and this should be "can keep".

Reply:
Bah, sorry this was my mistake, will fix asap